docs/source/techspecs/luareference.rst

```diff
@@ -1974,6 +1974,9 @@ field:set_value(value)
     Set the value of the I/O port field.  For digital fields, the value is
     compared to zero to determine whether the field should be active; for
     analog fields, the value must be right-aligned and in the correct range.
+field:clear_value()
+    Clear programmatic override from the value and restore it to its original
+    state.
```

Comment:
“programmatically overridden”

Also, original state isn’t quite right, it restores regular behaviour, but the actual value may well be different on the next update if it’s mapped to an absolute control.